lib/presentations.js

```diff
@@ -20,13 +20,17 @@ export async function sign({
 
   // pick a suite from the accepted proof types
   let suite;
-  if(acceptedProofTypes) {
+
+  if(Array.isArray(acceptedProofTypes) && !!acceptedProofTypes.length) {
     for(const {name} of acceptedProofTypes) {
       suite = supportedSuites.get(name);
       if(suite) {
         break;
       }
     }
+    if(!suite) {
+      throw new Error(`Unsupported proof types, ${acceptedProofTypes}`);
```

Comment:
```suggestion
      throw new Error(`No accepted proof type (${acceptedProofTypes.join(', ')} is supported.`);
```

Reply:
After adding the check to see if signer type is included in `acceptedProofTypes` [8081c7d](https://github.com/digitalbazaar/bedrock-web-wallet/pull/33/commits/8081c7d5aa3c741ac08513a185a4fbf5edc66485), this becomes dead code. So, I've removed it.